solutions/standard/stack_definition.json

```diff
@@ -801,35 +805,187 @@
     },
     {
       "name": "watsonx_project_url",
-      "value": "ref:../Workload - Sample RAG App Configuration/outputs/watsonx_project_url"
+      "value": "ref:../members/Workload - Sample RAG App Configuration/outputs/watsonx_project_url"
     },
     {
       "name": "watsonx_project_id",
-      "value": "ref:../Workload - Sample RAG App Configuration/outputs/watsonx_project_id"
+      "value": "ref:../members/Workload - Sample RAG App Configuration/outputs/watsonx_project_id"
     },
     {
       "name": "watson_discovery_api_url",
-      "value": "ref:../Workload - Sample RAG App Configuration/outputs/watson_discovery_api_url"
+      "value": "ref:../members/Workload - Sample RAG App Configuration/outputs/watson_discovery_api_url"
     },
     {
       "name": "watson_discovery_project_id",
-      "value": "ref:../Workload - Sample RAG App Configuration/outputs/watson_discovery_project_id"
+      "value": "ref:../members/Workload - Sample RAG App Configuration/outputs/watson_discovery_project_id"
+    },
+    {
+      "name": "kms_instance_crn",
+      "value": "ref:../members/Essential Security - Encryption Key Management/outputs/kms_instance_crn"
+    },
+    {
+      "name": "kms_private_endpoint",
+      "value": "ref:../members/Essential Security - Encryption Key Management/outputs/kms_private_endpoint"
+    },
+    {
+      "name": "kms_public_endpoint",
+      "value": "ref:../members/Essential Security - Encryption Key Management/outputs/kms_public_endpoint"
+    },
+    {
+      "name": "event_notification_instance_crn",
+      "value": "ref:../members/Essential Security - Event Notifications/outputs/crn"
+    },
+    {
+      "name": "event_notification_instance_name",
+      "value": "ref:../members/Essential Security - Event Notifications/outputs/event_notification_instance_name"
+    },
+    {
+      "name": "event_notification_instance_guid",
+      "value": "ref:../members/Essential Security - Event Notifications/outputs/guid"
+    },
+    {
+      "name": "cloud_logs_crn",
+      "value": "ref:../members/Essential Security - Logging Monitoring Activity Tracker/outputs/cloud_logs_crn"
+    },
+    {
+      "name": "cloud_logs_guid",
+      "value": "ref:../members/Essential Security - Logging Monitoring Activity Tracker/outputs/cloud_logs_guid"
+    },
+    {
+      "name": "cloud_monitoring_crn",
+      "value": "ref:../members/Essential Security - Logging Monitoring Activity Tracker/outputs/cloud_monitoring_crn"
+    },
+    {
+      "name": "cloud_monitoring_guid",
+      "value": "ref:../members/Essential Security - Logging Monitoring Activity Tracker/outputs/cloud_monitoring_guid"
+    },
+    {
+      "name": "log_analysis_crn",
+      "value": "ref:../members/Essential Security - Logging Monitoring Activity Tracker/outputs/log_analysis_crn"
+    },
+    {
+      "name": "log_analysis_guid",
+      "value": "ref:../members/Essential Security - Logging Monitoring Activity Tracker/outputs/log_analysis_guid"
```

Comment:
remove `log_analysis_crn` and `log_analysis_guid` - the stack no longer deploys log analysis